dsn.go

```diff
@@ -138,80 +110,6 @@ func ParseDSN(dsn string) (*Config, error) {
 		conf.transactionIsolation = uint32(isolation)
 	}
 
-	if v := queries.Get("authentication"); v != "" {
-
```

Comment:
This should also not be removed because we want to allow people to set up authentication using the DSN.

Reply:
done